### PR TITLE
feat: added HtaQuery

### PR DESCRIPTION
**Pull request by AndyBuchwald**

### CODE REVIEW

File: history/metricq-history.js [hunk 1]
```diff
@@ -108,6 +108,52 @@ class AnalyzeQuery {
...
+      metric
+    )
+
+    return this
+  }
+
+  _parse_result (result) {
+    let data = result['data']
+    for(const metric_data of Object.values(data)) {
+      for (const value of metric_data[metric_data.mode]) {
```
Comment (bmario) [inline history/metricq-history.js:131]:
```suggestion
      for (const value of metric_data[metric_data.mode]) {
```

How does this even work? `value` is modified.

File: history/metricq-history.js [hunk 2]
```diff
@@ -108,6 +108,52 @@ class AnalyzeQuery {
...
+          'from': this.from.toISOString(),
+          'to': this.to.toISOString()
+        },
+        'maxDataPoints': this.points,
+        'metrics': this.metrics
+      }
+      axios.post(`${this.mq.url}/query_hta`, paramters, this.mq.config).then(result =>
+        resolve(this._parse_result(result))
+      ).catch(error => reject(error)
+      )
```
Comment (bmario) [inline history/metricq-history.js:152]: The formatting looks strange.

**Review by bmario — CHANGES_REQUESTED** [review]

File: history/metricq-history.js (hunk 1 above)
Comment (AndyBuchwald) [reply history/metricq-history.js:131]: Only value.time gets modified. const only stops reassigning value itself.

File: history/metricq-history.js (hunk 2 above)
Comment (AndyBuchwald) [reply history/metricq-history.js:152]: ?
The formatting is the same as in query and analyze.

Comment (bmario) [reply history/metricq-history.js:152]: we'll fix the format in a different commit / pr

**Review by bmario — APPROVED** [review]

**Pull request merged**